### ISSUE TITLE
Orange Metal Catalyst recipe broken preventing Hydrazine/mars/venus.

### ISSUE THREAD

**Issue by Pyriana** [jmoiron/quantum-skies#21]
Adv rockets appear to be broken.

Unable to produce Orange Metal Catalyst

Machine is HV recipe is HV, machine has power, and programmed circuit is 32 in the machine (also tried the native one on the machine) tried every number just in case. This prevents the dense hydrazine mixed fuel used to make the advanced rocket parts

regular rocket also always crashes any player who isn't the person who launches the rocket on the server.

![image](https://github.com/user-attachments/assets/a4d986ac-a95d-4492-9c81-9a329f3a89aa)

**Comment by jmoiron** [comment]
Was able to "fix" the catalyst recipe in dev w/ KubeJS.  Opened a fix for the upstream mod as well.

The rocket thing sounds like I'll have to report it to GCYR upstream and see if screret can help fix it, because I'm not experienced enough with the intricacies of modding for multiplayer.  Pretty sure the mod has not seen a lot of multiplayer testing.

Can you give me some more information on what is crashing?  Is it a rocket with 2 seats crashing another passenger? Is it any kind of rocket?

FWIW, there is a rebalance coming in the next few days with an 0.4.0 version number that pulls tungsten forward onto the moon. This will push out the venus/mars gating to the assline/netherstars, which will ease the requirement on producing the better rocket fuels before you are fully into late EV (eg. w/ RTM coils and EV energy hatches).

**Comment by jmoiron** [comment]
Okay, instead of accepting the simple fix for the catalyst, the whole catalyst system has been reworked GT-- `0.1.3`

If you want to fix for your current world, you can add this to `kubejs/server_scripts/gtnn.js`:

```js
    event.remove({id: "gtceu:mixer/orange_metal_catalyst"})
    greg.mixer("orange_metal_catalyst")
        .circuit(32)
        .itemInputs("gtceu:vanadium_dust")
        .itemInputs("gtceu:palladium_dust")
        .itemOutputs("64x gtceu:orange_metal_catalyst_dust")
        .EUt(GTValues.VA[GTValues.HV])
        .duration(20*8);
```

**Comment by Pyriana** [comment]
> Was able to "fix" the catalyst recipe in dev w/ KubeJS. Opened a fix for the upstream mod as well.

I ended up putting a creative chest with the catalyst till it gets fixed which is good enough for me, it was never going to be the bottleneck for anything from what I can tell.

> The rocket thing sounds like I'll have to report it to GCYR upstream and see if screret can help fix it, because I'm not experienced enough with the intricacies of modding for multiplayer. Pretty sure the mod has not seen a lot of multiplayer testing.
> 
> Can you give me some more information on what is crashing? Is it a rocket with 2 seats crashing another passenger? Is it any kind of rocket?

For the rocket thing it's either with 2 seats or just existing in the same plane, if we're on different planes it's fine, ie if they go to the moon while I'm at the main base, it crashes, if they come back from the moon while I'm at the main base it doesn't.

Happens every time, if you can direct me to where a crash report would be I can include that. It hard crashes minecraft all the way back to the launcher.


> FWIW, there is a rebalance coming in the next few days with an 0.4.0 version number that pulls tungsten forward onto the moon. This will push out the venus/mars gating to the assline/netherstars, which will ease the requirement on producing the better rocket fuels before you are fully into late EV (eg. w/ RTM coils and EV energy hatches).

We've just come back from mars last night with backpacks full of mars regolith so we're likely going to be decent enough on tungsten till we can build an me dimensional anchor thing and a core driller (which we needed tungsten for the main machine part)

As an aside we can't figure out how to make the multiblock steam turbine work when we put a stainless steel rotor in it it says the efficiency is -1% and we can't find any videos or info on what we're doing wrong.

This modpack has been keeping our attention really well though, better than most of the packs we've played, we just increased our nitro benzene production 4x to keep up with power as we start introducing ev machines into our automation.

**Comment by jmoiron** [comment]
> I ended up putting a creative chest with the catalyst till it gets fixed which is good enough for me, it was never going to be the bottleneck for anything from what I can tell.

The fix for this is out now with 0.4.0.  The catalysts now have 100 uses and I think they're meant to go into a special catalyst hatch.

> For the rocket thing it's either with 2 seats or just existing in the same plane, if we're on different planes it's fine, ie if they go to the moon while I'm at the main base, it crashes, if they come back from the moon while I'm at the main base it doesn't.
> Happens every time, if you can direct me to where a crash report would be I can include that. It hard crashes minecraft all the way back to the launcher.

You can open an issue on the [GCYR github repo](https://github.com/Argent-Matter/gcyr) with the crash report.  I want to look into this myself a little too but since it requires another player it might take me some time to organize that.

> As an aside we can't figure out how to make the multiblock steam turbine work when we put a stainless steel rotor in it it says the efficiency is -1% and we can't find any videos or info on what we're doing wrong.

Not aware of this; there shouldn't be any modifications to the steam turbine though, so this could be a gtceu modern issue.

> This modpack has been keeping our attention really well though, better than most of the packs we've played

<3 thanks! :D 

> we just increased our nitro benzene production 4x to keep up with power as we start introducing ev machines into our automation.

The church of benzene!  I usually go petrol but I went benzene on my recent playtest (I'm slightly behind where you are, in EV prepping for the next rocket) and it's pretty nice.  Would love to see some base pics eventually!

I'll leave this issue open as a reminder to follow up on the gcyr and turbine issues.
